src/deployment/data_migration.py

```diff
@@ -38,8 +42,38 @@ def migrate_task_os(table_service: TableService) -> None:
     print("migrated %s rows" % count)
 
 
+def migrate_notification_keys(table_service: TableService) -> None:
```

Comment:
We should maybe have much more explicit names and/or comments, including version info, for migration functions.

Reply:
Something to think about for the future.